### PR TITLE
Add Cycles Views to Swagger, APIv2

### PR DESCRIPTION
**Pull request by Myoldmopar**
#### Any background context you want to provide?
The cycles API wasn't yet in Swagger/APIv2 land.  

#### What's this PR do?

* Move cycles to API v2
* Expose cycles in Swagger
* Ignores the cycles endpoints in the trailing slash unit test because it seems it doesn't matter...

#### How should this be manually tested?

* Make sure endpoints show up on Swagger
* Make sure endpoints work on Swagger
* Unit tests should pass
* Monkey test front end to find unexpected issues

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/133876307

### CODE REVIEW

**Comment by Myoldmopar** [comment]
Some things I need to work out:
 - [ ] Making sure security is covered
 - [ ] Monkey testing front end
 - [ ] Double check on js services

File: seed/static/seed/js/services/project_service.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ angular.module('BE.seed.service.project', [])
         var defer = $q.defer();
         $http({
             method: 'GET',
-            url: '/api/v2/projects-count/',
+            url: '/api/v2/projects/count/',
```
Comment (nllong) [inline seed/static/seed/js/services/project_service.js]: nice catch

File: seed/views/cycles.py [hunk 2]
```diff
@@ -4,60 +4,187 @@
...
+        cycles = Cycle.objects.filter(
+            organization_id=org_id_in_query, pk=pk
+        )
+        if cycles.exists():
+            cycle = cycles[0]
+        else:
+            return JsonResponse({
+                'status': 'error',
+                'message': 'Could not access cycle with id={}'.format(pk)
+            }, status=status.HTTP_403_FORBIDDEN)
```
Comment (nllong) [inline seed/views/cycles.py]: vs 401? I'm fine with 403, just don't know which we should use in general.

Comment (nllong) [inline seed/views/cycles.py]: nvm: Receiving a 403 response is the server telling you, “I’m sorry. I know who you are–I believe who you say you are–but you just don’t have permission to access this resource. Maybe if you ask the system administrator nicely, you’ll get permission. But please don’t bother me again until your predicament changes.”

File: seed/views/cycles.py [hunk 3]
```diff
@@ -4,60 +4,187 @@
...
+        """
+
+        body = request.data
+        org_id = int(request.query_params.get('organization_id', None))
+        try:
+            org = Organization.objects.get(pk=org_id)
+        except Organization.DoesNotExist:
+            return JsonResponse({"status": 'error', 'message': 'organization_id not provided'},
+                                status=status.HTTP_400_BAD_REQUEST)
+        record = Cycle.objects.create(
```
Comment (nllong) [inline seed/views/cycles.py]: what if body['name'] body['start'] or body['end'] are undefined/missing?

File: seed/views/cycles.py [hunk 4]
```diff
@@ -4,60 +4,187 @@
...
+              description: The organization_id
+              required: true
+              paramType: query
+        """
+        body = request.data
+        organization_id = int(
+            request.query_params.get('organization_id', None))
+        cycle = Cycle.objects.filter(pk=pk, organization_id=organization_id)
+        cycle.update(
+            name=body['name'],
```
Comment (nllong) [inline seed/views/cycles.py]: ditto ... need to validate the existence of these values in the dict.

**Pull request merged**